tools/c7n_azure/c7n_azure/resources/automation_account.py

```diff
@@ -0,0 +1,78 @@
+from c7n_azure.resources.arm import ArmResourceManager
+from c7n_azure.provider import resources
+from c7n.filters import ValueFilter
+from c7n.utils import type_schema
+from c7n.filters.core import OPERATORS
+
+
+@resources.register('automation-account')
+class AutomationAccount(ArmResourceManager):
+    """Azure Account Automation Resource
+
+    :example:
+
+    This policy will lists the Automation Accounts within an Azure subscription
+
+    .. code-block:: yaml
+
+        policies:
+          - name: automation-account
+            resource: azure.automation-account
+    """
+
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Network']
```

Comment:
its not really Network re doc group

Reply:
Changed to `Integration`. I'm not quite sure whether it really can be interpreted as `Integration` kind of resource but other groups seem even less appropriate to me. I would suggest using `Management` group. But is there such one?
What do you think?